Yesod/Auth/OAuth2/Github.hs

```diff
@@ -47,12 +49,14 @@ instance FromJSON GithubUser where
     parseJSON _ = mzero
 
 data GithubUserEmail = GithubUserEmail
-    { githubUserEmail :: Text
+    { githubUserEmailEmail :: Text
```

Comment:
regarding this, the current name is crazy i agree, but i did that just so it would be consistent.

i thought that a better win would be to build up a better record structure like say - https://github.com/silky/yesod/blob/master/yesod-auth/Yesod/Auth/GoogleEmail2.hs#L470 - and then just solve this in that refactoring (i'm happy to make this change, but probs in a different PR).